### PR TITLE
assignment 3 first commit

### PR DESCRIPTION
**Pull request by spsparmar**

### CODE REVIEW

File: assignment3.yaml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  description: "The Search API is optimized to help you find the specific item you're looking for (e.g., a specific user, a specific file in a repository, etc.)."
+  version: "1.0.0"
+  title: "Git Hub Search"
+host: "developer.github.com"
+basePath: "/v3/search"
+tags:
+- name: "Repositories"
+  description: "Explore Search Repositories"
+schemes:
+- "http"
```
Comment (devanshutechracers) [inline assignment3.yaml]: http!!!!

**Review by devanshutechracers — CHANGES_REQUESTED** [review]
Changes are required in the submission. I have made the comments on the Pull request about the changes

**Review by devanshutechracers — APPROVED** [review]

File: assignment3.yaml [hunk 2]
```diff
@@ -0,0 +1,122 @@
+swagger: "2.0"
+info:
+  description: "The Search API is optimized to help you find the specific item you're looking for (e.g., a specific user, a specific file in a repository, etc.)."
+  version: "1.0.0"
```
Comment (rockon1985) [inline assignment3.yaml]: version 1?? Check the version of the github API and update accordingly!

**Review by rockon1985 — CHANGES_REQUESTED** [review]
Check the version of the github API and update accordingly

**Review by rockon1985 — APPROVED** [review]

File: assignment3.yaml [hunk 3]
```diff
@@ -0,0 +1,122 @@
+swagger: "2.0"
+info:
+  description: "The Search API is optimized to help you find the specific item you're looking for (e.g., a specific user, a specific file in a repository, etc.)."
+  version: "3"
```
Comment (devanshutechracers) [inline assignment3.yaml:4]: should be "v3" not just 3

File: assignment3.yaml [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+        in: "query"
+        required: false
+        type: "string"
+        items:
+          type: "string"
+          enum:
+          - "stars"
+          - "forks"
+          - "updated"
+      - name: "order"
```
Comment (devanshutechracers) [inline assignment3.yaml:45]: `description` property should be provided.

File: assignment3.yaml [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+        type: "string"
+        items:
+          type: "string"
+          enum:
+          - "fork"
+          - "in"
+          - "language"
+          - "size"
+        collectionFormat: "multi"
+      - name: "sort"
```
Comment (devanshutechracers) [inline assignment3.yaml:35]: `description` property should be provided.

File: assignment3.yaml [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+  /Repositories:
+    get:
+      tags:
+      - "Repositories"
+      summary: "Find Repositories"
+      description: "Returns Repositories"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "q"
```
Comment (devanshutechracers) [inline assignment3.yaml:23]: `description` property should be provided.

File: assignment3.yaml [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+        maximum: 1000
+        description: "Number of search results returns"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            type: "array"
+            items:
+             $ref: "#/definitions/Repositories"
+definitions:
```
Comment (devanshutechracers) [inline assignment3.yaml:68]: relevant examples are missing from definitions

**Pull request merged**